compose-builder/Makefile

```diff
@@ -172,9 +154,9 @@ ifeq (ds-onvif-camera, $(filter ds-onvif-camera,$(ARGS)))
 			TOKEN_LIST:=$(TOKEN_LIST),device-onvif-camera
 		endif
 		ifeq ($(KNOWN_SECRETS_LIST),)
-			KNOWN_SECRETS_LIST:=redisdb[device-onvif-camera],message-bus[device-onvif-camera]
+			KNOWN_SECRETS_LIST:=postgres[device-onvif-camera],message-bus[device-onvif-camera]
```

Comment:
Not sure if my understanding is correct here, device services don't access the database directly.

Therefore, the postgres secrets doesn't have to be generated for all the device services.

The old `redisdb` secret was used for the redis pub/sub for device services.